src/routes/registration.ts

```diff
@@ -67,7 +67,8 @@ export const registrationEndpoints = (
   router.post(
     GENERAL_PARTNER_CHOICE_URL,
     authentication,
-    dependencies.registrationController.redirectWithParameter()
+    // to be changed - use different method
```

Comment:
Again, when will this be changed? Can't it change now, before this PR is merged?